src/test/versionbits_tests.cpp

```diff
@@ -178,65 +178,40 @@ BOOST_FIXTURE_TEST_SUITE(versionbits_tests, TestingSetup)
 BOOST_AUTO_TEST_CASE(versionbits_test)
 {
     for (int i = 0; i < 64; i++) {
-        // DEFINED -> FAILED
```

Comment:
This change is from #19573. I believe it is dropped because BIP 8 specifies that the timeoutheight falls on a retarget block greater than the startheight, so this scenario is supposed to be impossible to occur. With BIP 9, it was possible because there was no guarantee of the number of blocks between the start time and timeout time, but with height, the number of blocks is guaranteed.